<launch>

  <node pkg="turtlesim" type="turtlesim_node" name="turtlesim_node" output="screen"/>

  <node pkg="naloga2" type="trj_service_node" name="traj_service_node"  output="screen">
    <param name="scale_linear" value="1" type="double"/>
    <param name="scale_angular" value="4" type="double"/>
    <param name="trajectory" value="square" type="string"/>
    <param name="duration" value="10" type="double"/>
    <remap from="cmd_vel" to="/turtle1/cmd_vel" />

  </node>

</launch>

